src/style.css

```
@@ -0,0 +1,36 @@
+body,
+h1,
+p,
+ul,
+li,
+main {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  position: absolute;
+  display: flex;
+  /* border: thick double #32a1ce; */
+  width: 140px;
+  height: 160px;
+  padding: 32px 22px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  /* background-color: black;
+  color: aqua; */
+}
+
+.stars__star {
+  position: relative;
+
+  left: auto;
+  /* border: thick double #32a1ce; */
+  content: "icon";
+  /* color: blue; */
```

```suggestion
  content: "";
```
Get rid of unnecessary properties, content can be an empty string